parlai/core/utils.py

```diff
@@ -960,22 +960,29 @@ def padded_tensor(items, pad_idx=0, use_cuda=False, left_padded=False):
     # max in time dimension
     t = max(lens)
 
+    # if input tensors are empty, we should expand to nulls
+    t = max(t, 1)
+
     if isinstance(items[0], torch.Tensor):
         # keep type of input tensors, they may already be cuda ones
         output = items[0].new(n, t)
     else:
         output = torch.LongTensor(n, t)
     output.fill_(pad_idx)
 
-    for i, item in enumerate(items):
+    for (i, item), length in zip(enumerate(items), lens):
```

Comment:
imho, `for i, (item, length) in enumerate(zip(items, lens))`

Reply:
ah yeah I like yours better too